src/main/scala/swiss/dasch/api/ProjectsEndpoints.scala

```diff
@@ -187,9 +187,13 @@ final case class ProjectsEndpoints(base: BaseEndpoints) {
     .out(header[String]("Content-Disposition"))
     .out(header[String]("Content-Type"))
     .out(streamBinaryBody(ZioStreams)(CodecFormat.OctetStream()))
+    .out(header[String]("Access-Control-Request-Origin"))
+    .out(header[String]("Access-Control-Request-Credentials"))
+    .out(header[String]("Access-Control-Request-Methods"))
+    .out(header[String]("Access-Control-Request-Headers"))
```

Comment:
Do we need all of them or you just trying? ;)

Reply:
I am imitating what Sipi is sending.